### ISSUE TITLE
Implement service point inventory page

### ISSUE THREAD

**Issue by cafootitt** [OpenSRP/opensrp-client-eusm#6]
Mockup: https://marvelapp.com/prototype/7d6gbea/screen/73387485
Mockup: https://marvelapp.com/prototype/1296hh24/screen/73418943
Mockup: https://marvelapp.com/prototype/1296hh24/screen/72966670
Mockup: https://marvelapp.com/prototype/1296hh24/screen/73366103

- [x] Back button takes user back to list view or map view, whichever one they came from
- [x] Service point icon, name, type, distance to user (or GPS unknown), and commune at the top
- [x] Service point check and GPS task are sorted to the bottom always if there is a product task in either section
- [x] Once tasks are completed, they move to the Checked section below, the image greys out and the green check mark appears
- [x] Before any tasks are completed, the text reads "No items checked yet". This disappears when the first task is completed.
- [x] After all tasks are completed, the text reads "No items to check".
- [x] Information displayed in each row:
-- Product image (if there is one, if no image, the generic product avatar displays)
-- Product name
-- Quantity (quantity doesn't display if quantity is blank)
-- Serial number (if no serial number, hide the "Serial #" text
- [x] When a product task is completed via the "flag problem" form, the yellow alert flag also appears for that completed task
- [x] The same yellow alert flag icon also displays when any Fix problem task is completed.

**Comment by cafootitt** [comment]
V0.0.1 - built on 10 Dec 2020 - lgtm, good work, some small UI clean up is moved to a new issue: #30
